charts/catalog/values.yaml

```diff
@@ -56,3 +56,6 @@ controllerManager:
   resyncInterval: 5m
   # Broker relist interval; format is a duration (`20m`, `1h`, etc)
   brokerRelistInterval: 24h
+  # Whether or not the controller supports a --broker-relist-interval flag. If this is 
+  # set to true, brokerRelistInterval will be used as the value for that flag
+  brokerRelistIntervalActivated: false
```

Comment:
I don't like that we have to do this, but the reasoning is sound. @arschles is there an issue for the above? LGTM